salt/metalk8s/orchestrate/downgrade/init.sls

```diff
@@ -64,6 +64,9 @@ Deploy node {{ node }}:
     - pillar:
         orchestrate:
           node_name: {{ node }}
+          {%- if pillar.get("orchestrate", {}).get("drain_timeout") %}
+          drain_timeout: {{ pillar.orchestrate.drain_timeout }}
+          {%- endif %}
```

Comment:
Why wouldn't we always pass this down, even if unset? Also, I find using the `pillar` execution module simpler than chaining `.get("key", {})`:

```suggestion
          drain_timeout: {{ salt.pillar.get("orchestrate:drain_timeout", default=0) }}
```

Reply:
Just used to, but of course, for nested dict it's really simpler to use pillar.get, I will change it